package.json

```diff
@@ -155,6 +155,56 @@
           "go"
         ],
         "startSessionCommand": "go.debug.startSession",
+        "configurationSnippets": [
+          {
+            "label": "Go: Launch package",
+            "description": "Debug the package in the program attribute",
+            "body":{
+              "name": "${2:Launch Package}",
+              "type": "go",
+              "request": "launch",
+              "mode": "debug",
+              "program": "^\"\\${workspaceRoot}${1:}\""
+            }
+          },
+          {
+            "label": "Go: Launch file",
+            "description": "Debug the file in the program attribute",
+            "body":{
+              "name": "${2:Launch file}",
+              "type": "go",
+              "request": "launch",
+              "mode": "debug",
+              "program": "^\"\\${workspaceRoot}${1:/main.go}\""
```

Comment:
good idea, this was a remnant of when I had combined debugggin package and file in which case one might want to remove the / as well